src/Symfony/Component/VarExporter/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+4.2.0
+-----
+
+ * added the component
```

Comment:
```patch
-  * added the component
+ * Created the component (extracted `PhpMarshaller` from the `Cache` component)
```

Could be valuable to give a hint of where it comes from.

Reply:
PhpMarshaller has never been released, better not give it any existence except in internal history
I'd keep as is :)